src/MoBi.Presentation/Tasks/ObservedDataTask.cs

```diff
@@ -151,11 +159,12 @@ private DimensionInfo.DataImporterSettings createDataImportSettings()
             IconName = ApplicationIcons.MoBi.IconName,
             Caption = $"{AppConstants.PRODUCT_NAME} - {AppConstants.Captions.ImportObservedData}"
          };
-         settings.AddNamingPatternMetaData(Constants.FILE);
+         addNamingPatterns(settings);
+         settings.NameOfMetaDataHoldingMolecularWeightInformation = AppConstants.Parameters.MOLECULAR_WEIGHT;
          return settings;
       }
 
-      private void adjustMolWeight(DataRepository observedData)
+      private void adjustMolWeightUnit(DataRepository observedData)
```

Comment:
hum..I am still confused. Why do you call this Unit. It is not setting the unit but really setting the value of molweight in the columns right? 
Is this not done in the importer now anyways?

Reply:
Again, you are right. The adjustment of the value according to the unit is being done in Core, and since in Core we are removing the MolWeight Extended property of the DataRepository in the end anyway, the function would actually in practice always return with after the first check. 

I have removed this again.